README.md

```diff
@@ -129,19 +129,56 @@ DJANGO_WORKER_COUNT=1 DJANGO_WORKER_INDEX=0SPANNER_TEST_INSTANCE=django-tests ./
 
 ## Limitations
 
-Limitation|Comment|Resolution
----|---|---
-Lack of DEFAULT for columns|Cloud Spanner doesn't support using DEFAULT for columns thus the use of default values might have to enforced in your controller logic|
-Lack of FOREIGN KEY constraints|Cloud Spanner doesn't support foreign key constraints thus they have to be defined in code
-Lack of sequential and auto-assigned IDs|Cloud Spanner doesn't autogenerate IDs and this integration instead creates UUID4 to avoid [hotspotting](https://cloud.google.com/spanner/docs/schema-design#uuid_primary_key) so you SHOULD NOT rely on IDs being sorted|We generate UUID4s for each AutoField
-Numeric values are saved as FLOAT64|Cloud Spanner doesn't support the NUMERIC type thus we might have precision losses|Decimal and Numeric are translated to FLOAT64. If Cloud Spanner adds NUMERIC in the future, you might need to migrate your columns
-Optimistic transactions when running DDL and other statements|Cloud Spanner CANNOT run DDL (CREATE, DROP, ALTER) in a Transaction and CAN ONLY run them in the [DatabaseAdmin RPC]() so any mixes of DDL and other statements will make prior statements get run, DDL run, then following statements run separately|
-107 unexplored Django tests|We need to run every single test in the Django test suite for 100% certainty|Currently in progress to complete the [211 test cases](https://gist.github.com/odeke-em/d3476b6cb7c3485de6aa29956ed50a75#file-all_tests-txt)
-Table names CANNOT have spaces within them whether back-ticked or not|Cloud Spanner DOEST NOT support tables with spaces in them for example `Entity ID`|Ensure that your table names don't have spaces within them
-Table names CANNOT have punctuation marks and MUST contain valid UTF-8|Cloud Spanner DOEST NOT support punctuation marks e.g. periods ".", question marks "?" in table names|Ensure that your table names don't have punctuation marks
-Lack of stored procedures and triggers|Cloud Spanner DOES NOT support stored procedures nor triggers so if your code relies on them, you might need to move the logic into your application|Not supported
-Lack of VARIANCE, STDDEV database functions|Cloud Spanned doesn't yet implement these functions|
+### `AutoField` generates random IDs
+
+Spanner doesn't have a way to auto-generate primary key values. Instead,
+django-spanner monkeypatches `AutoField` to generate a random UUID4.
+
+To avoid [hotspotting](https://cloud.google.com/spanner/docs/schema-design#uuid_primary_key),
```

Comment:
I think that we need to add that:
>Please don't rely on incrementally increasing numeric ids when sorting.
>You'll need to rewrite your application logic if you rely to sorting by id.

Reply:
I'm not convinced. I think programmers know how random values differ from serial values.

Reply:
They do but let’s err on the side of caution and explicitness, so that someone reading this guide can just add items to their checklist. I’ve had to call out this fact many times when describing implications of switching backends to customers, and this was to very technical engineers.